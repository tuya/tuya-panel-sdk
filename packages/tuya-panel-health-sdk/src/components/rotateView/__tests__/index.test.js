import React from 'react';
import { View } from 'react-native';
import { shallow } from 'enzyme';
import RotateView from '../index';

describe('RotateView components', () => {
  it('basic render', () => {
    const wrapper = shallow(
      <RotateView active>
        <View style={{ width: 200, height: 200 }} />
      </RotateView>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
