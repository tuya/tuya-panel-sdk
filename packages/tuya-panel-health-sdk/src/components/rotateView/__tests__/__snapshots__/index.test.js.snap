// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RotateView components basic render 1`] = `
<AnimatedComponent
  style={
    Array [
      undefined,
      Object {
        "transform": Array [
          Object {
            "rotate": "0deg",
          },
        ],
      },
    ]
  }
>
  <View
    style={
      Object {
        "height": 200,
        "width": 200,
      }
    }
  />
</AnimatedComponent>
`;
